console/servlets/pom.xml

```diff
@@ -49,23 +56,60 @@
         </executions>
         <configuration>
           <excludes combine.children="append">
-            <!-- TODO: Is this correct? I'd suggest to auto-download jquery and jquery-ui and include that to the output as part of the build -->
-            <!--
-            <dependency>
-                <groupId>org.webjars</groupId>
-                <artifactId>jquery-ui</artifactId>
-                <version>1.11.4</version>
-            </dependency>
-            <dependency>
-                <groupId>org.webjars</groupId>
-                <artifactId>jquery</artifactId>
-                <version>1.11.2</version>
-            </dependency>
-            -->
+            <!-- TODO: Still got one unlicensed file in there ... -->
```

Comment:
Please update the top level source LICENSE file to reflect only what's now bundled in the source.
Presumably the WAR's included LICENSE continues to require noting that it bundles d3,jquery,...?